spec/models/article_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  it 'will not create without a title' do
+    described_class.create({ body: 'This is a body....' })
+    expect(described_class.count).to eq 0
+  end
+
+  it 'will not create without a body' do
+    article = described_class.create({ title: 'Post #1' })
+    expect(article.valid?).to be false
+  end
+
+  it 'will not create article with less that 50 chars in body' do
+    article = described_class.create({ title: 'Post #1', body: 'Too short...' })
+    expect(article.valid?).to be(false)
+  end
```

Comment:
Also consider more cases for boundary conditions e.g. minimum limit ±1 character, max. limit ± 1 character